pkg/project/project.go

```diff
@@ -5,7 +5,7 @@ var (
 	gitSHA      = "n/a"
 	name        = "cluster-operator"
 	source      = "https://github.com/giantswarm/cluster-operator"
-	version     = "2.2.1-dev"
+	version     = "2.2.1-njuettner"
```

Comment:
We need to revert this before merging.

Reply:
I'll run a test with the latest changes and if succeeding I will change it again.